frontend/store/session/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+import type { RootState } from 'store';
+import { SignIn } from 'types/sign-in';
+
+// Define the initial state using that type
+const initialState: SignIn = {
+  email: undefined,
+  password: undefined,
+};
+
+export const sessionSlice = createSlice({
```

Comment:
If you do decide to keep using Redux, I would suggest that you add a comment above this line to explain what this slice is used for (and on which pages).